docs/source/v2.1/WaterCycle/index.rst

```diff
@@ -0,0 +1,16 @@
+E3SMv2.1 (Water Cycle)
+====================
+
+The simulation compaign for E3SMv2 (Water Cycle) was performed with two
+separate model configurations:
+
+* **v2_1.LR** (lower resolution)
+
+If you use data from this simulation campaign, please cite the relevant overview
+manuscripts.
```

Comment:
Please add Smith et al. 2024, GMD as the overview reference paper
https://gmd.copernicus.org/preprints/gmd-2024-149/